.github/workflows/ci.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           GO_EXECUTABLE_PATH=$(which go)
           sudo $GO_EXECUTABLE_PATH test -v -run "TestFetchImageSignaturesDockerPublic" ./launcher
       - name: Run specific tests to capture potential data race
-        run: go test ./launcher/agent -race -run TestCacheConcurrentSetGet
+        run: go test -v ./launcher/... -race -run 'TestCacheConcurrentSetGet|^.*TokenRefresh.*$|^TestFetchContainerImageSignatures'
```

Comment:
Given the recent concurrency issues, we should just run everything in launcher. The perf hit is only for testing/CI and it's orders of magnitudes faster than the image build/test anyway

https://go.dev/doc/articles/race_detector#Runtime_Overheads

Reply:
Done.